### PR TITLE
Adding upgrade Indexer to glossary

### PR DESCRIPTION
**Pull request by MichaelMacaulay**

### CODE REVIEW

File: website/pages/en/glossary.mdx [hunk 1]
```diff
@@ -24,6 +24,8 @@ title: Glossary
 
 - **Indexer's Self Stake**: The amount of GRT that Indexers stake to participate in the decentralized network. The minimum is 100,000 GRT, and there is no upper limit.
 
+- **upgrade Indexer**: a temporary Indexer designed to act as a fallback for subgraph queries not serviced by other Indexers on the network. It ensures a seamless transition for subgraphs upgrading from the hosted service by readily serving their queries upon being published. The upgrade Indexer is not competitive with other Indexers, and supports chains that were previously exclusive to the hosted service.
```
Comment (benface) [inline website/pages/en/glossary.mdx:27]:
```suggestion
- **Upgrade Indexer**: a temporary Indexer designed to act as a fallback for subgraph queries not serviced by other Indexers on the network. It ensures a seamless transition for subgraphs upgrading from the hosted service by readily serving their queries upon being published. The upgrade Indexer is not competitive with other Indexers, and supports chains that were previously exclusive to the hosted service.
```

**Comment by MichaelMacaulay** [comment]
@benface Thank you for the suggestion! Got that committed - would you be able to approve this?

File: website/pages/en/glossary.mdx [hunk 2]
```diff
@@ -24,6 +24,8 @@ title: Glossary
 
 - **Indexer's Self Stake**: The amount of GRT that Indexers stake to participate in the decentralized network. The minimum is 100,000 GRT, and there is no upper limit.
 
+- **Upgrade Indexer**: a temporary Indexer designed to act as a fallback for subgraph queries not serviced by other Indexers on the network. It ensures a seamless transition for subgraphs upgrading from the hosted service by readily serving their queries upon being published. The upgrade Indexer is not competitive with other Indexers, and supports chains that were previously exclusive to the hosted service.
```
Comment (benface) [inline website/pages/en/glossary.mdx:27]:
```suggestion
- **Upgrade Indexer**: A temporary Indexer designed to act as a fallback for subgraph queries not serviced by other Indexers on the network. It ensures a seamless transition for subgraphs upgrading from the hosted service by readily serving their queries upon being published. The upgrade Indexer is not competitive with other Indexers, and supports chains that were previously exclusive to the hosted service.
```

**Comment by benface** [comment]
@MichaelMacaulay – Sorry, just left another comment about something that I didn't notice the first time. 😅 Consider it approved once that's fixed!

**Pull request merged**